reference/specs/bindings/twitter.md

```diff
@@ -1,23 +1,24 @@
 # Kubernetes Events Binding Spec
```

Comment:
Title for this topic is wrong (although not in this PR)

Reply:
done